packages/source-fetcher/lib/etherscan.ts

```diff
@@ -167,7 +173,10 @@ const EtherscanFetcher: FetcherConstructor = class EtherscanFetcher
     return {
       contractName: result.ContractName,
       sources: {
-        [filename]: result.SourceCode
+        //we prepend this header comment so that line numbers in the debugger
+        //will match up with what's displayed on the website; note that other
+        //cases don't display a similar header on the website
+        [filename]: etherscanCommentHeader + result.SourceCode
```

Comment:
Note that this is probably better style:
```suggestion
        [filename]: `${etherscanCommentHeader}${result.SourceCode}`
```

Reply:
Eh, for a simple concatenation like this I'm not going to bother unless you really think it's important... which evidently you don't as you approved. :)